tutorials/6_Advanced_ChangeInputs.md

```diff
@@ -44,3 +44,21 @@ Adding New Input Data
 
 The input data for REMIND are generated by using the R-libraries *mrremind* (https://github.com/pik-piam/mrremind) and *madrat* (https://github.com/pik-piam/madrat). While *mrremind* contains all calculations tailored to REMIND-input-data, the package *madrat* provides the general wrapper functions and helpful tools. For further information read the vignettes part of the R-packages.
 
+How to Update Input Data
+========================
+
+
+1. Check for the current input data revision number in this cluster folder: `/p/projects/rd3mod/inputdata/cache`. Alternatively, run the helper tool `lastRev` (`/p/projects/rd3mod/tools/lastrev`) to get a list of the last five revX.XXX*_remind.tgz items in the default moinput output directory.
+2. Clone the repo `https://gitlab.pik-potsdam.de/REMIND/preprocessing-remind` to your tmp folder on the cluster and edit the `start.R` file by inserting the next revision number. If an old revision number is used, the input data will not be recalculated. Input data for a new regional resolution will be recalculated based on the existing cache information.
+
+3. Start the script with 
+
+``` bash
+sbatch slurm_start.sh
+```
+
+The .log file lists the progress and potential errors. This process might take a while (currently >8 hours).
+
+4. If the process terminates without errors, do a test run with the new input data. To do this, clone the REMIND repo and update the data input version using your recently created data revision number in the `config/default.cfg` file and run one scenario (e.g. SSP2EU-Base). 
+5. If the test run completes without errors, update `cfg$revision` in `config/default.cfg` and push the change to the REMIND repo, so REMIND uses the new data by default.
```

Comment:
Edit: I just checked my latest run on the cluster, and it appears that this is overwritten automatically once you start a run with info from the cfg file

Reply:
ok, good to know, thanks. Still I would suggest to add this change in main.gms to the PR if updating the input revision.

Reply:
i just noticed that this line refers to a comment, not an actual command: https://github.com/remindmodel/remind/blob/develop/main.gms#L86. 
however, i can see why updating the comment might help to avoid confusion. could you ask @LaviniaBaumstark about the best way to proceed and adjust the tutorial if she agrees with you? merging this first version for now.